docs/guide/hello-world.md

```diff
@@ -85,14 +83,14 @@ import page from 'page';
  */
 import { makeLayout, render as clientRender } from 'controller';
 import { navigation, siteSelection } from 'my-sites/controller';
-import { helloWorld } from './controller';
+import Controller from './controller';
```

Comment:
Thanks for the update @jsliang! Now that you've made adjustments in the controller example, I think we can revert this (and the call bellow) to original usage.